test/fixtures/wallet-web/static/config/credentialDisplayData.json

```diff
@@ -0,0 +1,636 @@
+{
+  "PermanentResidentCard": {
+    "id": [
+      "$.id"
+    ],
+    "brandColor": "boatBlue",
+    "issuanceDate": [
+      "$.issuanceDate"
+    ],
+    "properties": {
+      "image": {
+        "path": [
+          "$.credentialSubject.image"
+        ],
+        "label": "Photo",
+        "type": "image",
+        "format": ""
+      },
+      "givenName": {
+        "path": [
+          "$.credentialSubject.givenName"
+        ],
+        "label": "Given Name"
+      },
+      "familyName": {
+        "path": [
+          "$.credentialSubject.familyName"
+        ],
+        "label": "Family Name"
+      },
+      "gender": {
+        "path": [
+          "$.credentialSubject.gender"
+        ],
+        "label": "Gender"
+      },
+      "birthDate": {
+        "path": [
+          "$.credentialSubject.birthDate"
+        ],
+        "label": "Date of birth",
+        "type": "date",
+        "format": "yyyy-mm-dd"
+      },
+      "birthCountry": {
+        "path": [
+          "$.credentialSubject.birthCountry"
+        ],
+        "label": "Country of Birth"
+      },
+      "residentSince": {
+        "path": [
+          "$.credentialSubject.residentSince"
+        ],
+        "label": "Resident Since",
+        "type": "date",
+        "format": "yyyy-mm-dd"
+      }
+    },
+    "icon": "credential--uscis-icon.svg",
```

Comment:
We need to add code to load credential images from external source. At the moment, it looks like these are bundled into code base.

Reply:
I agree